packages/zoltan2/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -2007,6 +2003,10 @@ void AlgMJ<mj_scalar_t, mj_lno_t, mj_gno_t, mj_part_t, mj_node_t>::
                 = part_size;
             });
 
+            auto local_new_coordinate_permutations =
+              this->new_coordinate_permutations;
+            auto local_coordinate_permutations =
+              this->coordinate_permutations;
```

Comment:
You don't need the `parallel_for` below; just use `Kokkos::deep_copy`.

Reply:
Fixed.